scripts/generate.v3.ts

```diff
@@ -30,6 +30,219 @@ const ethereumDocumentConfig = {
+      signature: signatureForInvalidDocument,
+    },
+  };
+  writeFileSync(
+    "./test/fixtures/v3/dnsdid-revocation-store-invalid-signed.json",
+    JSON.stringify(signedInvalidDnsDidDocument, null, 2)
+  );
+
+  info("Revoking document...");
+  const cmdRevoke = `oa document-store revoke -h 0x${signedDnsDidDocument.proof.merkleRoot} -a ${ethereumDocumentConfig.documentStore} -k ${ethereumDocumentConfig.wallet.key} -n ${ethereumDocumentConfig.network}`;
+  execSync(cmdRevoke, { timeout: ethereumDocumentConfig.timeout });
+};
+**/
+
+const generateDidRevocationStore = async () => {
+  info("Generating DID-Revocation-Store files");
+  const baseDidDocumentRevocationStore = {
+    ...baseDidDocument,
+    openAttestationMetadata: {
+      ...baseDidDocument.openAttestationMetadata,
+      revocation: {
+        type: v3.RevocationType.RevocationStore,
+        address: ethereumDocumentConfig.documentStore,
+      },
+    },
+  };
+  writeFileSync(
+    "./test/fixtures/v3/did-revocation-store.json",
+    JSON.stringify(baseDidDocumentRevocationStore, null, 2)
+  );
+  const wrappedBaseDidDocumentRevocationStore = await __unsafe__use__it__at__your__own__risks__wrapDocument(
+    baseDidDocumentRevocationStore
+  );
+  writeFileSync(
+    "./test/fixtures/v3/did-revocation-store-wrapped.json",
+    JSON.stringify(wrappedBaseDidDocumentRevocationStore, null, 2)
+  );
+  const { merkleRoot } = wrappedBaseDidDocumentRevocationStore.proof;
+  const signature = await wallet.signMessage(utils.arrayify(`0x${merkleRoot}`));
+  const signedDidDocument: v3.SignedWrappedDocument = {
+    ...wrappedBaseDidDocumentRevocationStore,
+    proof: {
+      ...wrappedBaseDidDocumentRevocationStore.proof,
+      key: "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
+      signature,
+    },
+  };
+  writeFileSync(
+    "./test/fixtures/v3/did-revocation-store-signed.json",
+    JSON.stringify(signedDidDocument, null, 2)
+  );
+  const validSignatureWithoutDnsTxt = {
+    ...baseDidDocument,
+    openAttestationMetadata: {
+      ...baseDidDocument.openAttestationMetadata,
+      identityProof: {
+        type: v3.IdentityProofType.DNSDid,
+        identifier: "notinuse.tradetrust.io",
+      },
+    },
+  };
+  const wrappedInvalidDnsDidDocument = await __unsafe__use__it__at__your__own__risks__wrapDocument(
+    validSignatureWithoutDnsTxt
+  );
+  const signatureForInvalidDocument = await wallet.signMessage(
+    utils.arrayify(`0x${wrappedInvalidDnsDidDocument.proof.merkleRoot}`)
+  );
+  const signedInvalidDnsDidDocument: v3.SignedWrappedDocument = {
+    ...wrappedInvalidDnsDidDocument,
+    proof: {
+      ...wrappedInvalidDnsDidDocument.proof,
+      key: "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
+      signature: signatureForInvalidDocument,
+    },
+  };
+  writeFileSync(
+    "./test/fixtures/v3/did-revocation-store-invalid-signed.json",
+    JSON.stringify(signedInvalidDnsDidDocument, null, 2)
+  );
```

Comment:
Probably can leave these out. You probably want two files only:

1. DID document with revocation store that is not revoked
1. DID document with revocation store that is revoked

Reply:
sure